src/physics/types.rs

```diff
@@ -142,6 +142,26 @@ impl Field {
     }
 }
 
+
+// Invironment ///////////////////////////////////////////////////////////////
+
+pub struct Invironment {
+
+    pub bodies: HashMap<u32, Body>,
+    pub fields: HashMap<u32, Field>,
+}
+
+impl Invironment {
+    pub fn update(&mut self) {
+
+        // update each body
+        for (id, field) in self.fields.iter_mut() {
+            
+        }
+    }
+
```

Comment:
Try to keep the style consistent with the rest of the file. You should delete empty lines after method bodies. The same goes for before and after struct properties.